repository/category.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<site>
+   <feature id="org.eclipse.pde" version="0.0.0"/>
+   <feature id="org.eclipse.pde.source" version="0.0.0"/>
+   <feature id="org.eclipse.pde.spies" version="0.0.0"/>
+   <feature id="org.eclipse.pde.spies.source" version="0.0.0"/>
+   <feature id="org.eclipse.pde.unittest.junit" version="0.0.0"/>
+   <feature id="org.eclipse.pde.unittest.junit.source" version="0.0.0"/>
+   <!-- Tests -->
+   <bundle id="org.eclipse.pde.ua.tests"/>
```

Comment:
do we need the tests in the site? if yes can we probably move them to a dedicated category?